### DISCUSSION TITLE
Error when trying to create a docusaurus app from a git repository template on Windows

### DISCUSSION THREAD

**Discussion by cbarillet** [facebook/docusaurus#10789] [Q&A]
Hello everyone.  
  
I am new on docusaurus and I try to create my first docusaurus web site.
I try to create my docusaurus site from a git repository: https://github.com/PaloAltoNetworks/docusaurus-openapi-docs.  
  
When I try to follow the steps of the installation with my Windows 11 laptop, I have an error:
![image](https://github.com/user-attachments/assets/5573a0a4-0da5-475b-aff2-109f93c22f7d)  
  
or   
![image](https://github.com/user-attachments/assets/d78860b8-e89a-4a84-8d95-f6613641b768)  
  
I tried a lot of different scenarios on Windows:

1. Run the command with git bash
2. Run the command with cmd
3. Run the command with powershell as admin
4. Run the command with powershell as normal user 

I have the same error whatever the way I run the command line.

I was able to unblock me by running my command on my Macbook: everything went well. 
![image](https://github.com/user-attachments/assets/587447b6-5ca1-489c-8168-4622744d0454)
 
  
I open this discussion just to help the community because without my Macbook I didn't find the solution and everyone doesn't have another system with him. Can you help me to find the correct solution to do what I tried to do with Windows 11 please ? Or maybe it is a bug ?

**Comment by slorber** [comment]
I'm not sure what is happening

I confirm this works on macos `npx create-docusaurus@3.6.3 my-website2 https://github.com/PaloAltoNetworks/docusaurus-openapi-docs.git`

I don't currently have Windows to reproduce the problem

Moving from shelljs to execa might fix the problem:
https://github.com/facebook/docusaurus/pull/10358

The problematic part is likely in `escapeShellArg` here:

```ts
  logger.info('Creating new Docusaurus project...');

  if (source.type === 'git') {
    const gitCommand = await getGitCommand(source.strategy);
    const gitCloneCommand = `${gitCommand} ${escapeShellArg(
      source.url,
    )} ${escapeShellArg(dest)}`;
    if (shell.exec(gitCloneCommand).code !== 0) {
      logger.error`Cloning Git template failed!`;
      process.exit(1);
    }
    if (source.strategy === 'copy') {
      await fs.remove(path.join(dest, '.git'));
    }
  }
```
